hmt_escrow/eth_bridge.py

```diff
@@ -6,7 +6,7 @@
 from web3 import Web3, HTTPProvider, EthereumTesterProvider
 from web3.contract import Contract
 from web3.middleware import geth_poa_middleware
-from hmt_escrow.kvstore_abi import abi as kvstore_abi
```

Comment:
All these should be put back since I remember we have upstream issues with relative imports

Reply:
Using non-relative import here is an error and should not work, as it would use a different version of htm_escrow installed with pip or pipenv, not the one in the code.

As I don't have that module installed (since I'm working on it locally), it can't be imported, and that looks to be the correct behavior. So if that doesn't work on upstream that is probably a bug which should be fixed actually.
But maybe it does works, as the tests pass with the relative import.

Reply:
I tried a couple things, and I found out that if I use the project folder as root instead of the python code folder, the non-relative import works also.